packages/client/src/internals/getAbortController.ts

```diff
@@ -8,11 +8,10 @@ export function getAbortController(
     return customAbortControllerImpl;
   }
 
-  // eslint-disable-next-line @typescript-eslint/prefer-optional-chain
   if (typeof window !== 'undefined' && window.AbortController) {
     return window.AbortController;
   }
-  // eslint-disable-next-line @typescript-eslint/prefer-optional-chain
```

Comment:
???

Reply:
`pnpm lint-fix` did that, i undoed the changes and a workflow applied formatting once again 😵‍💫